src/sentry/auth/access.py

```diff
@@ -246,10 +246,17 @@ def has_any_project_scope(self, project: Project, scopes: Collection[str]) -> bo
             return True
 
         if self.member and features.has("organizations:team-roles", self.member.organization):
-            for team in project.teams.all():
-                membership = self._team_memberships.get(team)
-                if membership is None:
-                    continue
+            with sentry_sdk.start_span(op="check_access_for_all_project_teams") as span:
```

Comment:
Should we adopt a prefix for this `op` name? I'm following `"get_project_access_in_teams"` above, but I don't think that's a precedent.

I'm also pretty sure no one's actively monitoring that one. I could change both names, or just delete the old one?

Reply:
Just double-checking, is `op` `operation`? I've seen it littered through our code base

Reply:
I believe that's what it stands for, yes. In this context, I read it as the unique name for the operation being measured.